### PR TITLE
Update ember version to 3.10

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,32 +18,35 @@ env:
...
+    - stage: "Additional Tests"
+      env: EMBER_TRY_SCENARIO=ember-lts-2.18
+    - env: EMBER_TRY_SCENARIO=ember-lts-3.4
+    - env: EMBER_TRY_SCENARIO=ember-release
+    - env: EMBER_TRY_SCENARIO=ember-beta
+    - env: EMBER_TRY_SCENARIO=ember-canary
+    - env: EMBER_TRY_SCENARIO=ember-default-with-jquery
 
-after_success:
-  - bash <(curl -s https://codecov.io/bash)
```
Comment (anehx) [inline .travis.yml]: This is missing

**Review by anehx — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (velrest) [reply .travis.yml]: Fixed :+1:

File: .eslintrc.js [hunk 2]
```diff
@@ -1,15 +1,52 @@
...
+    ecmaVersion: 2018,
     sourceType: 'module'
   },
+  plugins: [
+    'ember'
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended'
+  ],
```
Comment (anehx) [inline .eslintrc.js:13]: Prettier is missing, also don't forget to run `yarn lint:js --fix` after adding it in the config..

**Review by anehx — CHANGES_REQUESTED** [review]
Prettier

**Pull request closed without merging**